cmake/fftw_on_windows.cmake

```diff
@@ -39,13 +39,34 @@ CheckExitCodeAndExitIfError("tar")
 
 get_filename_component(_vs_bin_path "${CMAKE_LINKER}" DIRECTORY)
 
-execute_process(
-    COMMAND "${_vs_bin_path}/lib.exe" ${LIBEXE_64} /def:${CMAKE_BINARY_DIR}/fftw/libfftw3-3.def /out:${CMAKE_BINARY_DIR}/fftw/libfftw3-3.lib
-    WORKING_DIRECTORY "${CMAKE_BINARY_DIR}/fftw"
-    OUTPUT_VARIABLE OutVar
-    ERROR_VARIABLE ErrVar
-    RESULT_VARIABLE ExitCode)
-CheckExitCodeAndExitIfError("lib.exe: ${OutVar} ${ErrVar}")
+if(CMAKE_HOST_UNIX AND WIN32)
```

Comment:
This change is unrelated

Reply:
mm suppose I've did some mistake when duplicating branch let me think how to remove this from request..